consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/network/ValidatorPeers.java

```diff
@@ -75,6 +75,11 @@ public void send(final MessageData message, final Collection<Address> blackList)
 
   private void sendMessageToSpecificAddresses(
       final Collection<Address> recipients, final MessageData message) {
+    LOG.trace(
+        "Sending message to peers messageCode={} messageData={} recipients={}",
```

Comment:
the message data contained might be useful - but I suspect that given its a byte array, our ability to deconstruct it might be ... hard.